app/src/main/java/com/cmput301f18t20/medicalphotorecord/Photo.java

```diff
@@ -13,25 +13,62 @@
 package com.cmput301f18t20.medicalphotorecord;
 
 import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.util.Base64;
+import android.util.Log;
+
+import java.io.ByteArrayOutputStream;
 
 import Exceptions.PhotoTooLargeException;
 
 public class Photo {
-    protected String name;
-    protected String directory;
-    protected byte[] byteimage = null; /* likely will need to convert to byte array for storage in elasticsearch */
-    Bitmap bitmap;
+
+    private String recordUUID;
```

Comment:
Do we also need a Photo UUID to uniquely identify them?

Reply:
They'll likely also need the UserID of the account they're associated with as we'll need to fetch all photos associated with a User when they log in.

Reply:
do we need to fetch all photos for user? the requirement only said to show photos for specific record (if we assume that all record uuid are unique even if they're from different patient)

Reply:
I added photo uuid to photo model class

Reply:
I think we need to see all the photos since we need to load all photos into a slideshow in ViewProblemActivity